lib/_http_client.js

```diff
@@ -241,11 +241,9 @@ function ClientRequest(input, options, cb) {
   const headersArray = ArrayIsArray(options.headers);
   if (!headersArray) {
     if (options.headers) {
-      const keys = ObjectKeys(options.headers);
       // Retain for(;;) loop for performance reasons
       // Refs: https://github.com/nodejs/node/pull/30958
```

Comment:
^ This is a relevant comment here – we’ll generally want to avoid performance and behavioural regressions. I’d guess that `for in` is a bit slower, and additionally, it’s not only listing own properties like `ObjectKeys` does.

Reply:
Yeah, you’re right. I realized there’re some different between Keys and for in